docs/reference/src/main/asciidoc/events.asciidoc

```diff
@@ -376,13 +376,17 @@ Weld defines the following non-portable notification options:
 [cols=",,",options="header",]
 |=======================================================================
 |Key |Value type|Description
-|`org.jboss.weld.notification.mode` |`String`|The notification mode. Possible values are: `SERIAL` (default), `PARALLEL`. See also <<notification-modes-table,Notification modes>>.
+|`weld.async.notification.mode` |`String`|The notification mode. Possible values are: `SERIAL` (default), `PARALLEL`. See also <<notification-modes-table,Notification modes>>.
+|`weld.async.notification.timeout` |`Long`|The notification timeout (in milliseconds) after which the returned completion stage must be completed. If the time expires the stage is completed exceptionally with a `CompletionException` holding the `java.util.concurrent.TimeoutException` as its cause. The expiration does not abort the notification of the observers.
```

Comment:
The input can be a `String` as well (e.g. when they load this pram from some config file), shouldn't we mention it as well?